src/Monolog/Handler/MongoDBHandler.php

```diff
@@ -44,8 +50,12 @@ protected function write(array $record)
     {
         if ($this->mongoCollection instanceof \MongoDB\Collection) {
             $this->mongoCollection->insertOne($record["formatted"]);
+        } else if($this->mongoCollection instanceof \MongoCollection) {
+            $this->mongoCollection->insert($record["formatted"]);
```

Comment:
Rather than `else if`, I'd suggest a series of `if` blocks that return:

```php
if ($this->mongoCollection instanceof \MongoDB\Collection) {
    $this->mongoCollection->insertOne($record['formatted']);

    return;
}

// $this->manager instanceof  \MongoDB\Driver\Manager
if ($this->mongoCollection instanceof \MongoCollection) {
    $this->mongoCollection->insert($record['formatted']);

    return;
}

$bulk = new \MongoDB\Driver\BulkWrite;
$bulk->insert($record['formatted']);
$this->$manager->executeBulkWrite($this->namespace, $bulk);
```